sympy/stats/tests/test_rv.py

```diff
@@ -112,8 +112,8 @@ def test_H():
     L = Geometric('L', p)
     X = Normal('X', 0, 1)
     D = Die('D', sides = 4)
-    assert H(X, X < 0) == -log(2) + 1 + log(pi)
-    assert H(X, X > 0, X < 1) == -log(2) + 1 + log(pi)
+    assert H(X, X > 0) == -log(2)/2 + 1/2 + log(pi)/2
+    assert H(X, X <= 0) == -log(2)/2 + 1/2 + log(pi)/2
```

Comment:
This is essentially a repetition. Perhaps something like `H(X, X > 1)` would not look too complicated.